### ISSUE TITLE
move from sqlite-manager to another format for the database source files

### ISSUE THREAD

**Issue by GoogleCodeExporter** [De7vID/klingon-assistant#35]
```
Currently, the import from XML to the SQL database is dependent on 
sqlite-manager:
http://code.google.com/p/sqlite-manager/

This runs on top of Firefox, meaning that the import process is dependent on 
two pieces of somewhat unstable software.

The database should be moved to more established and stable technology.
```

Original issue reported on code.google.com by `De.vID.jonpIn` on 14 Jan 2013 at 8:21

**Comment by GoogleCodeExporter** [comment]
```
Another major problem with sqlite-manager is that it breaks silently on errors. 
For example, if there are two "notes" columns in one entry the second one will 
simply override the first without warning.
```

Original comment by `De.vID.jonpIn` on 15 Jan 2013 at 7:49

**Comment by GoogleCodeExporter** [comment]
```
Script added to do the conversion using command line tools:
http://code.google.com/p/klingon-assistant/source/detail?r=3ca08f7cee5b240cf4f51
1c8de2826c80281f523
```

Original comment by `De.vID.jonpIn` on 16 Jan 2013 at 8:02
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed as of revision 710e7e73aafd. The script will even catch common xml errors.
```

Original comment by `De.vID.jonpIn` on 17 Jan 2013 at 10:40
- Changed state: **Fixed**
